co-simulation/tools/geneate_background_traffic.sh

```diff
@@ -60,12 +60,12 @@ vehicle_classes=(
   "vehicle.audi.tt"
   "vehicle.jeep.wrangler_rubicon"
   "vehicle.chevrolet.impala"
-  "vehicle.mini.cooper_s"
-  "vehicle.mercedes.coupe"
+  "vehicle.mini.cooperst"
+  "vehicle.mercedes-benz.coupe"
   "vehicle.bmw.grandtourer"
   "vehicle.citroen.c3"
-  "vehicle.ford.mustang"
-  "vehicle.lincoln.mkz_2017"
+  "vehicle.mustang.mustang"
+  "vehicle.lincoln.mkz2017"
   "vehicle.seat.leon"
   "vehicle.nissan.patrol"
   "vehicle.nissan.micra"
```

Comment:
The script's name has a typo can we fix it under this PR? 
Where does this get called? can we make sure it is updated there too
otherwise looks good.